src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -1597,6 +1597,9 @@ namespace DataHeatBalance {
         Real64 OABalanceFanElec = 0.0;       // Fan Electricity {W} due to OA air balance
         Real64 SumEnthalpyM = 0.0;           // Zone sum of EnthalpyM
         Real64 SumEnthalpyH = 0.0;           // Zone sum of EnthalpyH
+        // Mechanical ventilation
+        Real64 ZoneOAMassFlow = 0.0; // zone mech vent mass flow rate {kg/s}
+        Real64 ZoneOAMass = 0.0;     // zone mech vent total mass for time {kg}
```

Comment:
Two of these have been moved to `state.dataHeatBal->ZnAirRp` in commit https://github.com/NREL/EnergyPlus/pull/9519/commits/76cbe0c6fc7bca96fe4ac440ae235b7942649fb8 If this approach looks reasonable, then I'll proceed with moving more of these.